### PR TITLE
IA Pages - load different templates on the static page based on the dev milestone

### PR DESCRIPTION
**Pull request by MariagraziaAlastra**
WIP, will add Handlebars templates afterwards; @jagtalon will help, so he's going to make some PRs to this branch.
//cc @russellholt

### CODE REVIEW

**Comment by jagtalon** [comment]
@MariagraziaAlastra Added more space for the title and added the live label:

screen shot 2015-02-05 at 2 13 50 pm

**Comment by MariagraziaAlastra** [comment]
@jagtalon Awesome, thanks!

**Comment by jagtalon** [comment]
It should probably align with the "Enable edit mode" though

**Comment by MariagraziaAlastra** [comment]
@jagtalon Actually not, for now let's leave it like this, there's a lot of stuff going on on the top right corner and I fear aligning the IA name with that would clump a bit the view. Anyway we can always get back to that :)

**Comment by jagtalon** [comment]
@MariagraziaAlastra oh yeah def that's why I didn't do anything :)

**Comment by jagtalon** [comment]
@MariagraziaAlastra about the "Safe Search"--it's more of a "code" thing than a "metadata" thing isn't it? That is, toggling this would mean changing the behavior of https://duckduckgo.com/?q=ud+brb&kp=-1&ia=dictionary and not just the data about it.

I think I would expect it to change an IA from "Safe" to "Unsafe" and vise versa which we wouldn't want to do, right?

**Comment by MariagraziaAlastra** [comment]
@jagtalon No, that's not what's going to do, it's just informational. Like, does it require the safe search to be off or not? And then the developer will code accordingly :)

**Comment by jagtalon** [comment]
@MariagraziaAlastra oh right because it's the planning stage! Thanks

File: src/ia/js/IAPage.js [hunk 1]
```diff
@@ -45,24 +45,53 @@
                 $.getJSON("/ia/view/" + DDH_iaid + "/json", function(ia_data) {
 
                     // Show latest edits for admins and users with edit permissions
-                    var x = {};
+                    var latest_edits_data = {};
                     if (ia_data.edited) {
-                        x = DDH.IAPage.prototype.updateData(ia_data, x, true);
-                   } else {
-                        x = ia_data.live;
+                        latest_edits_data = DDH.IAPage.prototype.updateData(ia_data, latest_edits_data, true);
```
Comment (russellholt) [inline src/ia/js/IAPage.js]: calling a function on the prototype seems kinda weird. No `new` ?

**Comment by MariagraziaAlastra** [comment]
Still WIP, but on a good track. The functionality is there, the layout is there (thanks @jagtalon), we just need to decide exactly what fields to add and where, and then add them, of course :)

Dev page for users without permissions:
screen shot 2015-02-06 at 15 04 06


Dev page for admins (same thing for users with edit permissions, only without the "Mark as complete" button):
screen shot 2015-02-06 at 15 04 21

**Comment by jagtalon** [comment]
@MariagraziaAlastra not sure if this is for this PR, but I think we should go with the repo being a dropdown instead.

For the Perl Module--would that be useful? What if there are multiple ones like Dictionary?

**Comment by MariagraziaAlastra** [comment]
@jagtalon I guess that only the triggering one should be included there, since that's how we are distinguishing proper IAs from auxiliary submodules.

Yeah a dropdown would be better for that, not sure if I'm going to do that in this PR though

**Comment by jagtalon** [comment]
@MariagraziaAlastra okay thanks we can do it in another PR. :) Hmm, yeah it's tricky like when someone makes a PR with multiple triggering submodules like SeatGeek. I guess we could just treat that as different IAs

**Comment by MariagraziaAlastra** [comment]
@jagtalon Yup :)

**Comment by jagtalon** [comment]
@MariagraziaAlastra Maybe "Triggers" would be useful in the dev phase.

**Comment by MariagraziaAlastra** [comment]
Now the repo has a link to GitHub and there's also a link to the docs next to it :)
I wonder if we can link to the docs for other fields as well? Uhm.
screen shot 2015-02-06 at 16 40 04

@jagtalon Yeah triggers should definitely be in planning.

**Comment by MariagraziaAlastra** [comment]
screen shot 2015-02-09 at 23 02 54

**Comment by MariagraziaAlastra** [comment]
Final version (for now) - ready for review. Will add the new fields to the DB and make improvements in other PRs.
screen shot 2015-02-10 at 15 48 30

**Comment by jagtalon** [comment]
:+1: Nice!

**Pull request merged**